io/io/inc/TFile.h

```diff
@@ -186,7 +187,7 @@ class TFile : public TDirectoryFile {
    enum EFileType { kDefault = 0, kLocal = 1, kNet = 2, kWeb = 3, kFile = 4, kMerge = 5};
 
    TFile();
-   TFile(const char *fname, Option_t *option="", const char *ftitle="", Int_t compress=4);
+   TFile(const char *fname, Option_t *option="", const char *ftitle="", Int_t compress = ROOT::kUseGeneralPurposeCompressionSetting);
```

Comment:
The original value was 4, the new value is 1.

Reply:
humm .. right .. the original does not yet have the lz4 default revert .... nevermind ..